### PR TITLE
STY: flake8

### PR DESCRIPTION
**Pull request by Eric89GXL**
First test should fail because I haven't fixed any of the example files yet. Then it should pass.

### CODE REVIEW

File: examples/connectivity/plot_mne_inverse_connectivity_spectrum.py [hunk 1]
```diff
@@ -65,11 +66,10 @@
 fmin, fmax = 5., 40.
 sfreq = raw.info['sfreq']  # the sampling frequency
 
-con, freqs, times, n_epochs, n_tapers = spectral_connectivity(label_ts,
-        method='wpli2_debiased', mode='multitaper', sfreq=sfreq, fmin=fmin,
-        fmax=fmax, mt_adaptive=True, n_jobs=2)
+con, freqs, times, n_epochs, n_tapers = spectral_connectivity(
+    label_ts, method='wpli2_debiased', mode='multitaper', sfreq=sfreq,
+    fmin=fmin, fmax=fmax, mt_adaptive=True, n_jobs=2)
```
Comment (dengemann) [inline examples/connectivity/plot_mne_inverse_connectivity_spectrum.py]: I appreciate this style a lot, for multi line parameter matching it's the most efficient pep8 compatible idiom.

File: examples/decoding/plot_ems_filtering.py [hunk 2]
```diff
@@ -22,17 +22,19 @@
...
-print(__doc__)
+import matplotlib.pyplot as plt
 
 import mne
 from mne import io
 from mne.datasets import sample
 from mne.decoding import compute_ems
+
+print(__doc__)
+
```
Comment (dengemann) [inline examples/decoding/plot_ems_filtering.py]: why does the print doc go down, below the imports?

File: examples/connectivity/plot_cwt_sensor_connectivity.py [hunk 3]
```diff
@@ -12,20 +12,20 @@
...
-
 import numpy as np
+
 import mne
 from mne import io
 from mne.connectivity import spectral_connectivity, seed_target_indices
 from mne.datasets import sample
 from mne.time_frequency import AverageTFR
 
+print(__doc__)
```
Comment (agramfort) [inline examples/connectivity/plot_cwt_sensor_connectivity.py]: why did you change all these? flake8 does not complain for me.

File: examples/connectivity/plot_mne_inverse_label_connectivity.py [hunk 4]
```diff
@@ -106,7 +107,7 @@
     label_ypos.append(ypos)
 
 # Reorder the labels based on their location
-lh_labels = [label for (ypos, label) in sorted(zip(label_ypos, lh_labels))]
+lh_labels = [label for (yp, label) in sorted(zip(label_ypos, lh_labels))]
```
Comment (agramfort) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: yp is not even used?

File: examples/decoding/plot_decoding_csp_eeg.py [hunk 5]
```diff
@@ -75,8 +76,8 @@
 ###############################################################################
 # Classification with linear discrimant analysis
 
-from sklearn.lda import LDA
-from sklearn.cross_validation import ShuffleSplit
+from sklearn.lda import LDA  # noqa
```
Comment (agramfort) [inline examples/decoding/plot_decoding_csp_eeg.py]: what is this #noqa ? I get no warning with my flake8

File: examples/plot_channel_epochs_image.py [hunk 6]
```diff
@@ -56,8 +57,8 @@
 
 # and order with spectral reordering
 # If you don't have scikit-learn installed set order_func to None
-from sklearn.cluster.spectral import spectral_embedding
-from sklearn.metrics.pairwise import rbf_kernel
+from sklearn.cluster.spectral import spectral_embedding  # noqa
```
Comment (agramfort) [inline examples/plot_channel_epochs_image.py]: same remark

**Comment by agramfort** [comment]
besides LGTM

thanks @Eric89GXL for all this.

File: examples/connectivity/plot_mne_inverse_label_connectivity.py (hunk 4 above)
Comment (Eric89GXL) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: `ypos` is used later (or before?) so the variable name needed to be changed to avoid conflict

File: examples/connectivity/plot_cwt_sensor_connectivity.py (hunk 3 above)
Comment (Eric89GXL) [inline examples/connectivity/plot_cwt_sensor_connectivity.py]: your pep8 is too old; top-level imports not at top of file (the execution of the print statement makes the imports not at the top)

Comment (Eric89GXL) [inline examples/connectivity/plot_cwt_sensor_connectivity.py]: See the multitude of E402's from when I was in the middle of all these changes:

https://travis-ci.org/mne-tools/mne-python/jobs/51409195

File: examples/decoding/plot_decoding_csp_eeg.py (hunk 5 above)
Comment (Eric89GXL) [inline examples/decoding/plot_decoding_csp_eeg.py]: E402

File: examples/connectivity/plot_mne_inverse_label_connectivity.py (hunk 4 above)
Comment (agramfort) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: hum if it's used after your change will have a side effect I suppose

File: examples/decoding/plot_decoding_csp_eeg.py (hunk 5 above)
Comment (agramfort) [inline examples/decoding/plot_decoding_csp_eeg.py]: ok

File: examples/connectivity/plot_mne_inverse_label_connectivity.py (hunk 4 above)
Comment (agramfort) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: besides this one then the diff LGTM

Comment (Eric89GXL) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: No it's not used afterward in the way you think. It's one of these sorts of things (F812):
````
data = []
x = 0
a = [x for x in data]  # F812 flake error
```
If anything this change could prevented a future stupid error, but as it stands it does nothing other than avoid a flake error.

Comment (agramfort) [inline examples/connectivity/plot_mne_inverse_label_connectivity.py]: ok let's merge then !

thanks heaps !

**Comment by Eric89GXL** [comment]
Thanks for the eyes
